composer.json

```diff
@@ -30,7 +30,7 @@
     "phpunit/dbunit": "1.3.*",
     "phpunit/phpunit-story": "1.0.*",
     "phpunit/php-invoker": "1.1.*",
-    "ocramius/instantiator": "1.1.4"
+    "doctrine/instantiator": "1.1.4"
```

Comment:
Version 1.0.5 seems to be the newest available one. (Misleading message from composer.) Anyway with 1.0.5 the update works and the (basic) Solr 5.2.1 problems are resolved on CI-system. I am access the cores using "/solr/opus4-framework-fast" (for instance).